blocks/hotspot/hotspot.js

```diff
@@ -0,0 +1,53 @@
+export default function decorate(block) {
+  [...block.children].forEach((row, r) => {
+    if (r > 0) {
+      const content = [...row.children][0].textContent.trim();
+      const isImage = content.endsWith('.jpg') || content.endsWith('.png') || content.endsWith('.gif') || content.endsWith('.jpeg');
+      const isVideo = content.endsWith('.mp4') || content.endsWith('.webm') || content.endsWith('play') || content.endsWith('content/'); // Adjust condition as needed
```

Comment:
Same as previous comment.

Reply:
Any video URL can be incorporated as per the requirement. We can use also use regex as per requirement which is upto customisation.

Reply:
yeah, bit I think there may be cases where just looking at the url may not tell.  Maybe need a similar approach [like here](https://github.com/hlxsites/franklin-assets-selector/pull/107#discussion_r1777463410)